x-pack/test/case_api_integration/basic/tests/configure/get_connectors.ts

```diff
@@ -23,5 +30,54 @@ export default ({ getService }: FtrProviderContext): void => {
 
       expect(body).to.eql([]);
     });
+
+    it('should return the correct connectors', async () => {
+      const { body: connectorOne } = await supertest
+        .post('/api/action')
+        .set('kbn-xsrf', 'true')
+        .send(getServiceNowConnector())
+        .expect(200);
+
+      const { body: connectorTwo } = await supertest
```

Comment:
why test `'An email action'`?

Reply:
Because the endpoint returns only supported actions. Adding an unsupported action like `.email` ensures that the endpoint filters correctly.